06_puppy/n0npax/syncStore.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"errors"
+)
+
+func NewSyncStore() *SyncStore {
+	return &SyncStore{}
+}
+
+func (m *SyncStore) CreatePuppy(puppy *Puppy) int {
+	var newID int
+	m.Range(func(_, _ interface{}) bool {
+		newID++
+		return true
+	})
+	puppy.ID = newID
+	m.Store(puppy.ID, *puppy)
+	return puppy.ID
+}
+
+func (m *SyncStore) ReadPuppy(id int) (*Puppy, error) {
```

Comment:
Again can flip the logic and save a line here as well. Same as described in the memstore

Reply:
fixed. Thanks!